Mathlib/Geometry/Euclidean/Projection.lean

```diff
@@ -567,16 +567,14 @@ theorem dist_sq_eq_dist_orthogonalProjection_sq_add_dist_orthogonalProjection_sq
 variable {n : ℕ} [NeZero n] (s : Simplex ℝ P n)
 
 @[simp] lemma ne_orthogonalProjection_faceOpposite (i : Fin (n + 1)) :
```

Comment:
Arguably this lemma should be renamed to use `orthogonalProjectionSpan`, though of course then you'll need a deprecation.

bors d+

Reply:
(though if this will be changed again to use `altitude`, then fine to leave as is for now)

Reply:
Yes, I expect this to change to be about `altitudeFoot` (and thus move to go alongside that definition) (and I'm also doubtful of the need for deprecations for such recently added lemmas).

bors r+